subsystems/grippersubsystem.py

```diff
@@ -0,0 +1,97 @@
+# each roller alternates meaning if you were to pick up a cube you would eject a cone
+from enum import Enum, auto
+from commands2 import SubsystemBase
+from rev import SparkMaxLimitSwitch
+from wpilib import SmartDashboard
+
+from util.simneo import NEOBrushless
+
+import constants
+
+
+class GripperSubsystem(SubsystemBase):
+    class GripperState(Enum):
+        CubeIntake = auto()  # Cube gripper wheels, motor will move right
```

Comment:
Shouldn't this be just Intake and Outtake? We have two things we're doing: setting whether intaking or out taking, and setting whether we are doing things with cones or cubes